.github/workflows/itest.yml

```diff
@@ -11,11 +11,17 @@ jobs:
         with:
           go-version: 1.19.1
 
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
```

Comment:
Do we need the setup node action here as we install Node.js manually in https://github.com/csaf-poc/csaf_distribution/pull/349/files#diff-9e14b99a1dd4003f9986a20557f9e3ddbd8d071fc0d62d3e3eec7bf5bf2c0244

Reply:
Flagging @s-l-teichmann to answer the question...

Reply:
This was contributed by @bernhardreiter to make the integration test work again. Bernhard, what's the deal?

Reply:
The setup is improved to follow the original pattern:
 * .github/workflows/itest.yml  uses the github actions, which allow caching on the github runner's side.
 * docs/scripts/prepareUbuntuInstanceForITests.sh is used to prepare a naked Ubuntu system once. And is not part of the testing script itself. 
 
This allows the setup steps to be done only once and not per test run on a VM. It makes both the automatic and the manual tests faster (and saves a tiny bit of genery. ;) )